megamek/src/megamek/common/Mech.java

```diff
@@ -3246,6 +3270,34 @@ public TechAdvancement getConstructionTechAdvancement() {
                 .setPrototypeFactions(F_FW).setProductionFactions(F_FW)
                 .setAvailability(RATING_X, RATING_F, RATING_F, RATING_F)
                 .setStaticTechLevel(SimpleTechLevel.ADVANCED), //Superheavy industrial
+            //FIXME: The data is for Superheavy.
+            new TechAdvancement(TECH_BASE_IS).setISAdvancement(3060, 3076)
+                .setISApproximate(true, false).setTechRating(RATING_E)
+                .setPrototypeFactions(F_WB).setProductionFactions(F_WB)
+                .setAvailability(RATING_X, RATING_X, RATING_F, RATING_E)
+                .setStaticTechLevel(SimpleTechLevel.ADVANCED), //Superheavy command console
+            //FIXME: The data is for Small, except for Tech Level which follows command console.
+            new TechAdvancement(TECH_BASE_ALL).setISAdvancement(3060, 3067, 3080)
```

Comment:
The small cockpit tech advancement is more restrictive than the command console in all areas except static tech level, which you already adjusted. This FIXME can be removed as well.